libs/frontend/src/libs/products/components/ProductPhotos/useProductPhotos.ts

```diff
@@ -0,0 +1,25 @@
+import { gql } from '@apollo/client';
+import { useGetPhotosOnProductPhotosSuspenseQuery } from './__generated__/useProductPhotos';
+
+gql`
+  query GetPhotosOnProductPhotos($slug: String!) {
+    productBySlug(slug: $slug) {
+      screenshots {
+        imageUrl
+        imageAlt
+      }
+    }
```

Comment:
```suggestion
    productBySlug(slug: $slug) {
     id
      screenshots {
        imageUrl
        imageAlt
      }
    }
```
캐시 때문에 id 필드는 항상 추가해주세요